### DISCUSSION TITLE
Monitor - help!

### DISCUSSION THREAD

**Discussion by BigStevie** [picocomputer/community#86] [Q&A]
Newbie here - with a RP6502 built and displaying on VGA.  How do I connect to the 'monitor' though.  It appears to be different (i.e. shows 'color' (in color) in the banner at the top.   I have connected Minicom to /dev/ttyACM0 but that seems to be exactly the same as the VGA output.

Is there some magic incantation I need?

**Comment by rumbledethumps** [comment]
You need to manually build a very very very old version to see those deprecated and bogus boot messages.

**Reply by BigStevie** [reply]
Thanks - I did find that this is older code referenced in another thread and that now on boot it simply shows Picocomputer 6502 and VGA/RIA versions.

My real question - should the view on the Minicom monitor and the VGA monitor be 100% the same?   i.e. whatever I type on physical keyboard shows on Minicom and vice versa?
